SwSpotify/spotify.py

```diff
@@ -23,7 +28,13 @@ def find_spotify_uwp(hwnd, windows):
         text = win32gui.GetWindowText(hwnd)
         classname = win32gui.GetClassName(hwnd)
         if classname == "Chrome_WidgetWin_0" and len(text) > 0:
-            windows.append(text)
+            _, proc_id = win32process.GetWindowThreadProcessId(hwnd)
+            proc_h = win32api.OpenProcess(PROCESS_QUERY_LIMITED_INFORMATION, False, proc_id)
+            path = win32process.GetModuleFileNameEx(proc_h, None)
+            win32api.CloseHandle(proc_h)
+
+            if os.path.basename(path) == "Spotify.exe":
```

Comment:
this works but I prefer using pathlib over os.path for working with path names, since it's a lighter library and somewhat better API. I think it's just equivalent to Path(path).name so might be a good change